lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2483,49 +2484,45 @@ def quiver(self, *args,
                length=1, arrow_length_ratio=.3, pivot='tail', normalize=False,
                **kwargs):
         """
-        Plot a 3D field of arrows.
-
-        call signatures::
-
-            quiver(X, Y, Z, U, V, W, **kwargs)
-
-        Arguments:
+        ax.quiver(X, Y, Z, U, V, W, /, length=1, arrow_length_ratio=.3, pivot='tail', normalize=False, **kwargs)
```

Comment:
This replaces the signature in the docs, using the PEP570 syntax. There's precedent in ax.voxels, but it was admittedly set by me.

Reply:
All 6 arguments are required to `Axes3D.quiver`.

https://matplotlib.org/devel/documenting_mpl.html doesn't have any mention of "Call signatures" - perhaps it would be worth adding.

Reply:
If that doc signature is correct, we could directly change the signature of the function to be explicit.

Reply:
Until PEP570 is implemented, that would change the function signature to allow `X` to be passed by keyword. I think requiring it to be positional is a good idea.